main.go

```diff
@@ -133,9 +133,8 @@ Arguments:
 `,
 		Example: `  opkssh login
   opkssh login google
-  opkssh login --provider=<issuer>,<client_id>`,
+  opkssh login --provider=<issuer>,<client_id>,<client_secret>,<scopes>`,
```

Comment:
This might be debatable. We could also add an additional line instead, keeping the isser+client_id example as is. 
Happy to hear your thouhts what the most common use cases are.

Reply:
What do you mean by an additional line?